docs/running-e3sm-guide/guide-production.md

```diff
@@ -0,0 +1,97 @@
+
+## Running the Model
+
+To prepare for the long production simulation, edit the run_e3sm script and set `readonly run='production'`. In addition, you may need to customize some variables in the code block below to configure run options. Below is an example of how you might configure these variables:
+
+`# Production simulation`
+
+- `readonly PELAYOUT="L"`: 1=single processor, S=small, M=medium, L=large, X1=very large, X2=very very large. Production simulations typically use M or L. The size determines how many nodes will be used. The exact number of nodes will differ amongst machines.
+- `readonly WALLTIME="34:00:00"`: maximum wall clock time requested for the batch jobs.
+- `readonly STOP_OPTION="nyears"`: see next line
+- `readonly STOP_N="50"`: units and length of each segment (i.e., each batch job). E.g, the current configuration stops after 50 years.
+- `readonly REST_OPTION="nyears"`: see next line
+- `readonly REST_N="5"`: units and frequency for writing restart files (make sure `STOP_N` is a multiple of `REST_N`, otherwise the model will stop without writing a restart fie at the end). E.g., the current configurations saves restart files after every 5 years. 10 restart files will be saved, since `STOP_N=50`.
+- `readonly RESUBMIT=”9”`: number of resubmissions beyond the original segment. This simulation would run for a total of 500 years (=inital 50 + 9x50).
+- `readonly DO_SHORT_TERM_ARCHIVING=false`: leave set to false if you want to manually run the short term archive.
+
+Since the code had already been fetched and compiled for the [short tests](guide-prior-to-production.md), the toggle flags can be set to:
+
+```shell
+do_fetch_code=false
+do_create_newcase=true
+do_case_setup=true
+do_case_build=false
+do_case_submit=true
+```
+
+Finally, execute the script:
+
+```shell
+cd <run_scripts_dir>
+./run.<case_name>.sh
+```
+
+The script will automatically submit the first job. New jobs will be automatically be resubmitted at the end until the total number of segments have been run.
+
+## Looking at Results
+
+`ls <simulations_dir>/<case_name>` explanation of directories:
+
+- `build`: all the stuff to compile. The executable (`e3sm.exe`) is also there.
+- `case_scripts`: the files for your particular simulation.
+- `run`: where all the output will be. Most components (atmosphere, ocean, etc.) have their own log files. The coupler exchanges information between the components. The top level log file will be of the form `run/e3sm.log.*`.  Log prefixes correspond to components of the model:
+  - `atm`: atmosphere
+  - `cpl`: coupler
+  - `ice`: sea ice
+  - `lnd`: land
+  - `ocn`: ocean
+  - `rof`: river runoff
+
+Run `tail -f run/<component>.log.<latest log file>` to keep up with a log in real time.
+
+You can use the `sq` alias defined in [Useful Aliases](guide-appendix.md) to check on the status of the job. The `NODE` in the output indicates the number of nodes used and is dependent on the `processor_config` / `PELAYOUT` size.  
```

Comment:
Is it possible to link to specific subsections? E.g., specifically the "Useful Aliases" section of the Appendix?

Reply:
I think this is resolved?  The link to Useful Aliases is working.

Reply:
No, it only looks resolved because "Useful Aliases" is the very first subsection of the Appendix.